test/e2e/tests/onboarding/onboarding.spec.ts

```diff
@@ -27,16 +27,6 @@ import {
 import { switchToNetworkFlow } from '../../page-objects/flows/network.flow';
 
 describe('MetaMask onboarding', function () {
-  const ganacheOptions2 = {
-    accounts: [
-      {
-        secretKey:
-          '0x53CB0AB5226EEBF4D872113D98332C1555DC304443BEE1CF759D15798D3C55A9',
-        balance: convertToHexValue(10000000000000000000),
-      },
-    ],
-  };
-
```

Comment:
removing accounts option, as it  is not available in anvil, we can seed this specific account using setAccountBalance below